server/api/strategicgroups/index.ts

```diff
@@ -191,6 +191,52 @@ export async function listProjects(req) {
   return { projectList: await dbprojects.getProjectsByGroup(req.body.id) };
 }
 
+export async function updateProject(req, body) {
+  const { project_id, project_name, project_url, project_license } = body;
+  // update project
+  await dbprojects.updateProject(
+    project_id,
+    project_name,
+    project_url,
+    project_license
+  );
+
+  // update groups
+  const groupList = body.project_groups.split(',').map(groupID => {
+    return parseInt(groupID, 10);
+  });
+  const projectGroups = await dbgroups.getGroupsByProjectId([project_id]);
+  for (const group of projectGroups) {
+    // clean up existing groups from project
+    if (groupList.indexOf(group.group_id) === -1) {
```

Comment:
The `includes` function might be more idiomatic, but this is ok too

Reply:
I am still so used to using indexOf but you are right, this should be includes. Changing.